onnx/defs/experiments/functions.cc

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "onnx/defs/function.h"
+using namespace ONNX_NAMESPACE;
+
+static Common::Status BuildMVN(std::unique_ptr<FunctionProto>* func_proto) {
+  if (nullptr == func_proto) {
+    return Status(
+        Common::CHECKER,
+        Common::INVALID_ARGUMENT,
+        "func_proto should not be nullptr.");
+  }
+
+  func_proto->reset(new FunctionProto);
+  auto& func = **func_proto;
+  func.set_name("FuncMeanVarianceNormalization");
+  func.set_doc_string(
+      "A MeanVarianceNormalization Function: Perform mean variance normalization "
+      "on the input tensor X using formula: <br/> ``` (X-EX)/sqrt(E(X-EX)^2) ``` <br/>"
+      "<b>INPUT: </b>X(float/float16/double) with Shape [N,C,W,H] <br/>"
+      "<b>ATTRIBUTE: </b><br/>&nbsp;&nbsp;&nbsp;&nbsp;axes: will be passed to ReducedMean Ops. "
+      "Use [0,2,3] for no across channel, [0,1,2,3] for across channel caculation.<br/>"
+      "&nbsp;&nbsp;&nbsp;&nbsp;(The KeepDims attribute in ReducedMean ops is set to true for caculation)<br/>"
+      "<b>OUTPUT: </b>X_MVN(float/float16/double) with Shape [N,C,W,H] <br/>");
+  func.set_since_version(8);
+  func.add_input("X");
+  func.add_output("X_MVN");
+  func.add_attribute("axes");
+  NodeProto* initial_node0 = func.add_node();
+  BuildNode(
+      initial_node0,
+      "Pow_exponent_0",
+      "",
+      "Initialize a Constant tensor to caculate squared products",
+      "Constant",
+      std::vector<std::string>{},
+      std::vector<std::string>{});
+  AttributeProto* value_attr = initial_node0->add_attribute();
+  value_attr->set_name("value");
+  value_attr->set_doc_string(
+      "Value for the constant tensor as power exponent to caculate MVN");
```

Comment:
A clearer description: Exponent (i.e., 2) to element-wisely calculate the square of a tensor